src/routes/v1/newsRoutes.js

```diff
@@ -1,21 +1,17 @@
 const express = require('express');
 const { authenticateUser } = require('../../middlewares/authentication');
-// const { preferencesValidation } = require('../../middlewares/errorHandling');
+const { preferencesSchema } = require('../../models/user');
+const { commonValidationHandler } = require('../../middlewares/errorHandling');
 const newsController = require('../../controllers/newsController');
 
 const newsRouter = express.Router();
 
 newsRouter.get('/', authenticateUser, newsController.getAllNews);
```

Comment:
if authenticatedUser is used for all the routes than it can be called once before the route intialization instead of passing it into params

Reply:
updated as per comments